variables.tf

```diff
@@ -11,7 +11,7 @@ variable "environment" {
 variable "operator_version" {
   description = "Version of the Materialize operator to install"
   type        = string
-  default     = "v25.1.6"
+  default     = "v25.1.7"
```

Comment:
After this PR is merged, if we could release a new version of terraform-helm-materialize .... I can locally test the terraform-{cloud}-materialize PRs that have the helm version bump.

Reply:
Tagged as v0.1.12